components/filesystem.rst

```diff
@@ -313,6 +313,22 @@ contents at the end of some file::
 If either the file or its containing directory doesn't exist, this method
 creates them before appending the contents.
 
+``readFile``
+~~~~~~~~~~~~
+
+.. versionadded:: 7.1
```

Comment:
sometime I see this `versionadded` before sometime after the content it documents
is there a specific logic for one or another?

Reply:
I usually put it before the new feature contents ... because I think it's better to warn the user as soon as possible. But, many times the versionadded is displayed after the contents and works great too.

Reply:
Copy that